src/mongoDbFilter.ts

```diff
@@ -32,6 +32,10 @@ type GraphQLLeafFilter = GraphQLLeafFilterInner & {
     values?: any[];
 };
 
+function isLeafTypeTypeGuard(fieldFilter: GraphQLObjectFilter | GraphQLLeafFilter, fieldType: GraphQLObjectType | GraphQLLeafType): fieldFilter is GraphQLLeafFilter {
```

Comment:
Maybe put it in `common.ts`?

Reply:
I thought of it but it didn't look like there's anywhere else in need of that function.
but sure I can understand where that comes from, I'll push a refactor 👍